### ISSUE TITLE
Copy trait is derived when a struct contains __IncompleteArrayField

### ISSUE THREAD

**Issue by leeopop** [servo/rust-bindgen#458]
I have this error with generated code.

~~~~~~~~{.txt}
6948 |     pub field: __IncompleteArrayField<rte_acl_field>,
     |     ------------------------------------------------ this field does not implement `Copy`
~~~~~~~~

While the generated rte_acl_field is
~~~~~~~~{.rs}
#[repr(C)]
#[derive(Debug, Copy)]
pub struct rte_acl_field {
    pub value: rte_acl_field_types,
    pub mask_range: rte_acl_field_types,
}
~~~~~~~~

I think https://github.com/servo/rust-bindgen/commit/a00e8b03da825e13a809234abcace81d6cf59335
affects the result.

I check that 7d995d75e07798c1527d5ea3dd8a647ed79fe209 did not cause problems.
You may refer https://github.com/ANLAB-KAIST/rust-dpdk this repository to reproduce my result or put this usage scenario into test cases.

**Comment by emilio** [comment]
I added a commit fixing this in https://github.com/servo/rust-bindgen/pull/457, thanks for reporting it :)

**Comment by fitzgen** [comment]
Fixed in #457. Thanks!
